src/static/js/modules/util/web-storage-proxy.js

```diff
@@ -82,19 +91,24 @@ function setSessionType( sessionOnly ) {
  *   Use non-persistent storage (true) or persistent storage (false).
  * @returns {object} A local storage or session storage instance.
  */
-function _sessionOrLocal( sessionOnly ) {
+/* istanbul ignore next */
+function _storageType( storage ) {
```

Comment:
Good call, I was so set on the existing naming I didn't think that through all the way. Thanks